tools/isobuild/package-namespace.js

```diff
@@ -53,6 +53,8 @@ export class PackageNamespace {
    * will ONLY be bundled into production builds.
    * @param {Boolean} options.testOnly A package with this flag set to true
    * will ONLY be bundled as part of `meteor test`.
+   * @param {Boolean|String} options.deprecated A flag that will mark the
```

Comment:
@filipenevola This will be added to the documentation automatically, so there is no need to add anything else unless you want to me include some extra text to highlight it.